recaptcha/C001056.yaml

```diff
@@ -94,22 +80,17 @@ contact_form:
         selector: "#edit-submitted-address-administrative-area"
         value: Texas
         required: true
-    - javascript:
-      - name: Preview button
-        selectors: [ "input[type='submit'][value='Preview']" ]
-        commands: ["elements[0].disabled = false"]
-        required: true
-    - find:
-      - selector: input[type='submit'][value='Preview']
-    - click_on:
-      - selector: input[type='submit'][value='Preview']
     - find:
-      - selector: input[type='submit'][value='Submit']
+      - selector: "#edit-submitted-e-newsletter-opt-in-1"
     - click_on:
+      - value: Newsletter opt out 
+        selector: "#edit-submitted-e-newsletter-opt-in-1"
       - value: Submit
         selector: input[type='submit'][value='Submit'] 
-    - wait:
-      - value: 10
+    - find:
+      - selector: div.webform-confirmation
+    - find:
```

Comment:
Same q as above.

Reply:
Same as comment above (those this might be a little less important since I moved this yaml from /recaptcha to /members)